globals.go

```diff
@@ -272,6 +275,38 @@ const (
 	InvalidDivideInputError    = "$divide only supports numeric types"
 )
 
+var TaskStatuses = []string{
+	TaskStarted,
+	TaskSucceeded,
+	TaskFailed,
+	TaskSystemFailed,
+	TaskTestTimedOut,
+	TaskSetupFailed,
+	TaskAborted,
+	TaskStatusBlocked,
+	TaskStatusPending,
+	TaskKnownIssue,
+	TaskSystemUnresponse,
+	TaskSystemTimedOut,
+	TaskTimedOut,
+}
+
+// FilterValidTaskStatuses returns a slice of task statuses that are valid and are searchable.
+// It returns an empty array if emptyArrayForAll is true
+func FilterValidTaskStatuses(statuses []string, emptyArrayForAll bool) []string {
```

Comment:
Can this function not be in globals? Since it's just used by graphql it could be a private function in the util